.env.sample

```diff
@@ -1,9 +0,0 @@
-# RENAME THIS FILE TO .env
-
-# REPLACE THE GOOGLE CLIENT ID AND SECRET WITH YOUR OWN
-
-# URGENT TODO: Switch to non kapydev credentials
-
-GOOGLE_CLIENT_ID=blabla-blabla.apps.googleusercontent.com
-GOOGLE_CLIENT_SECRET=blabla-bla-bla
-SECRET_KEY=thisisnotaverysecretkeybutitsisokayfortesting
```

Comment:
We still need to keep this file for future users to set up their environment for testing

Reply:
Not sure why this was deleted... I added it back in